test/TestUtilities/Test.Utility/Signing/TrustedTestCert.cs

```diff
@@ -60,10 +63,18 @@ public class TrustedTestCert<T> : IDisposable
                 throw new InvalidOperationException($"The certificate used is valid for more than {maximumValidityPeriod}.");
             }
 #endif
-            StoreName = storeName;
-            StoreLocation = storeLocation;
-            AddCertificateToStore();
+            if (RuntimeEnvironmentHelper.IsMacOSX)
+            {
+                AddTrustedCert();
```

Comment:
naming: AddCertificateToMacStore maybe? Or to Machine store?

The method is doing an equivalent thing for Mac as AddCertificateToStore does for Windows/Linux. 
The names should communicate that.

Reply:
Thanks! Changed to `AddCertificateToStoreForMacOSX`